lib/Sema/CSDiagnostics.cpp

```diff
@@ -5863,12 +5863,16 @@ bool OutOfOrderArgumentFailure::diagnoseAsError() {
 
     // Move requires postfix comma only if argument is moved in-between
     // other arguments.
-    bool requiresComma =
-        !isExpr<BinaryExpr>(anchor) && PrevArgIdx != args->size() - 1;
+    std::string sign;
```

Comment:
Nitpick: I would call this `argumentSeparator` or something. But otherwise LGTM.

Reply:
Sure, it's all set to `argumentSeparator`.  Thanks